packages/cli/src/utils.ts

```diff
@@ -70,6 +70,16 @@ export async function parseConfigFilename(filename: string): Promise<string> {
   return filename;
 }
 
-export function ora(opts?: realOra.Options) {
+export function ora(opts?: realOra.Options): realOra.Ora {
   return realOra({ stream: process.stdout, ...opts });
 }
+
+/**
+ * leave a message and abort the process
+ * @param spinner spinner object
+ * @param message last message
+ */
+export function abort(spinner: realOra.Ora, message: string): void {
```

Comment:
May I ask you to make the abort in a class extends `Ora` so that we can just call `spinner.abort(msg)`?

Reply:
Or make this not depends on spinner, just using `console.error` to print the error message.